flake8_quotes/__init__.py

```diff
@@ -10,42 +10,40 @@ class QuoteChecker(object):
     name = __name__
     version = __version__
 
-    INLINE_QUOTES = {
-        # When user wants only single quotes
-        '\'': {
-            'good_single': '\'',
-            'good_multiline': '\'\'\'',
-            'bad_single': '"',
-            'bad_multiline': '"""',
-        },
-        '"': {
-            'good_single': '"',
-            'good_multiline': '"""',
-            'bad_single': '\'',
-            'bad_multiline': '\'\'\'',
-        },
-    }
+    INLINE_QUOTES = set(['"', "'"])
+    MULTILINE_QUOTES = set(['"""', "'''"])
 
     def __init__(self, tree, filename='(none)', builtins=None):
         self.filename = filename
 
     @classmethod
     def add_options(cls, parser):
+        # We don't care how it is exactly sorted, just that it is deterministic
```

Comment:
Now `optparse` requires the choices to be a list (`argparse` on the other hand doesn't), so I needed to convert it.

I personally prefer to use the correct iteration type (easily) available and here it is imho `set`: There is no inherent order and there are only lookup requests to it (apart from whatever `optparse` is doing) where `set` is the fastest. Granted this isn't really speed sensitive but if there is a reasonable type already there why not use it.

As you mentioned you want to re-review it next week I leave it for now and maybe just using a list is a reasonable compromise.